source/MaterialXGraphEditor/Graph.h

```diff
@@ -34,7 +34,7 @@ struct Link
 
 class Graph
 {
-  public:
+public:
```

Comment:
I see a few accidental edits to whitespace here and elsewhere, and it would be good to revert these before we merge the change.

Reply:
I think clang format file can be run this, can't it?